biolink/pieces-parts/rank-side-effects.rkt

```diff
@@ -39,13 +39,13 @@ returns: A list of 3 lists. These lists contain all of the drug curies resulting
                    ((D->O #f))
                    (D D->O O)))
   (define drugs (curies/query qtreats 'D))
-  ;;a query to get all of the symptoms/phenotypes of the given disease-curie
+  ;;a query to get all of the symptoms/phenotypes of the given disease-curie, + the disease-curie itself
```

Comment:
this reads a bit strange, what does the added stuff ... add?

Reply:
It allows the program to check to see if a drug is contraindicated for the disease-curie itself along with the symptoms that the disease causes.